src/component/handlers/edit/editOnCompositionStart.js

```diff
@@ -18,9 +18,10 @@ var EditorState = require('EditorState');
  * The user has begun using an IME input system. Switching to `composite` mode
  * allows handling composition input and disables other edit behavior.
  */
-function editOnCompositionStart(): void {
+function editOnCompositionStart(e): void {
   this.setRenderGuard();
   this.setMode('composite');
+  this._onCompositionStart(e);
```

Comment:
I think this private method might have problems if the class is munged. Also, if `compositionstart` is firing while we're in `EDIT` mode, then I don't think this function call has meaning.